test/framework/src/main/java/org/elasticsearch/repositories/blobstore/ESBlobStoreRepositoryIntegTestCase.java

```diff
@@ -81,13 +83,15 @@ protected Settings repositorySettings() {
         return Settings.builder().put("compress", randomBoolean()).build();
     }
 
-    protected final String createRepository(final String name) {
-        return createRepository(name, repositorySettings());
+    protected Settings repositorySettings(String repositoryName) {
+        return Settings.EMPTY;
     }
 
-    protected final String createRepository(final String name, final Settings settings) {
-        final boolean verify = randomBoolean();
+    protected final String createRepository(final String name) {
+        return createRepository(name, Settings.builder().put(repositorySettings()).put(repositorySettings(name)).build(), randomBoolean());
```

Comment:
I honored the original behavior, but you're right it should be `true` unless specified otherwise. I've made this change (verification `true` instead of `randomBoolean`).